runtime/lua/vim/lsp/callbacks.lua

```diff
@@ -102,6 +102,11 @@ M['textDocument/publishDiagnostics'] = function(_, _, result)
   vim.api.nvim_command("doautocmd User LspDiagnosticsChanged")
 end
 
+M['client/registerCapability'] = function(_, _, result, client_id)
```

Comment:
This needs to be implemented, but is necessary for pyright, the language server I'm debugging with.